cronjobs/src/commands/expire_orphan_attachments.py

```diff
@@ -71,5 +71,5 @@ def expire_orphan_attachments(event, context):
             print(
                 f"Marking orphan attachment gs://{STORAGE_BUCKET_NAME}/{blob.name} for deletion"
             )
-            blob.event_based_hold = False
+            blob.custom_time = datetime.now(timezone.utc)
```

Comment:
I presume you dobule-checked this actually takes a `datetime` object rather than, say, a Unix timestamp.

Reply:
yes!